src/ui/tabs/history_tab.py

```
@@ -962,82 +1003,82 @@ def handle_delete_selected_record(self):
+                month_val = get_csv_value(room_row, "Month", "").strip()
+                if month_val:
+                    key = month_val
+                    try:
+                        house = float(get_csv_value(room_row, "Total House Rent", "0"))
+                        water = float(get_csv_value(room_row, "Total Water Bill", "0"))
+                        gas = float(get_csv_value(room_row, "Total Gas Bill", "0"))
+                        unit = float(get_csv_value(room_row, "Total Room Unit Bill", "0"))
+                    except ValueError:
+                        continue
+                    grp = grouped.setdefault(key,{"house":0.0,"water":0.0,"gas":0.0,"unit":0.0})
+                    grp["house"]+=house; grp["water"]+=water; grp["gas"]+=gas; grp["unit"]+=unit
```

Ruff (0.11.9)

Multiple statements on one line (semicolon)

(E702)